distribution/docs/src/main/jdocs/content/_plugins/resource-usage-plugin-contents.adoc

```diff
@@ -0,0 +1,24 @@
+:type: plugin
+:status: published
+:title: Resource Usage Plugin
+:link: _resource_usage_plugin
+:plugintypes: preresource,postquery,postresource
+:summary: Monitors and limits system data usage.
+
+The Resource Usage Plugin monitors and limits data usage, and enables cancelling long-running queries.
```

Comment:
Agree. @ricklarsen, we should add a section for the `QueryMonitorPlugin`, which is a pre- and post-federated query plugin. The description should say something like _Monitors active queries which provides administrators a way to cancel long-running queries_. The rest of the information remains the same as this plugin.